CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# dbt_hubspot v0.17.1
+[PR #140](https://github.com/fivetran/dbt_hubspot/pull/140) includes the following updates:
+
+## Bug Fixes
+- Included explicit datatype casts to `{{ dbt.type_string() }}` within the join of `contact_merge_audit.vid_to_merge` to `contacts.contact_id` in the `int_hubspot__contact_merge_adjust` model.
+
+For context, the [Nov 2022 release of the Hubspot connector](https://fivetran.com/docs/connectors/applications/hubspot/changelog#november2022) deprecated the `contact_merge_audit` table in place of storing `calculated_merged_vids` in the `contact` table. Therefore if you do not have the `contact_merge_audit` table, the `vid_to_merge` is derived from the `calculated_merged_vids` in the `contact` table. 
```

Comment:
I still don't feel this additional context is all that necessary. November of 2022 was quite a long time ago and I bet we have the majority of customers on this new version. I don't feel we need to call this out in the CHANGELOG.